code/game/machinery/doors/airlock.dm

```diff
@@ -607,6 +607,28 @@
 	if(airlock_state == AIRLOCK_DENY)
 		update_icon(ALL, AIRLOCK_CLOSED)
 
+/obj/machinery/door/airlock/animation_length(animation)
+	switch(animation)
+		if(DOOR_OPENING_ANIMATION)
+			return 0.6 SECONDS
+		if(DOOR_CLOSING_ANIMATION)
+			return 0.6 SECONDS
+
+/obj/machinery/door/airlock/animation_segment_delay(animation)
+	switch(animation)
+		if(AIRLOCK_OPENING_TRANSPARENT)
+			return 0.1 SECONDS
+		if(AIRLOCK_OPENING_PASSABLE)
+			return 0.5 SECONDS
+		if(AIRLOCK_OPENING_FINISHED)
+			return 0.6 SECONDS
+		if("AIRLOCK_CLOSING_UNPASSABLE")
+			return 0.2 SECONDS
+		if("AIRLOCK_CLOSING_OPAQUE")
+			return 0.5 SECONDS
+		if("AIRLOCK_CLOSING_FINISHED")
+			return 0.6 SECONDS
```

Comment:
you're using the strings and not the defines here.

Reply:
I thought I got em all, fug